.circleci/config.yml

```diff
@@ -0,0 +1,132 @@
+version: 2
+
+variables:
+  restore_cache: &restore_cache
+    restore_cache:
+      keys:
+        - miniconda-{{ .Environment.MINICONDA_VER }}-{{ .Environment.BIOCONDA_UTILS_TAG }}-{{ arch }}
```

Comment:
Those `.Environment.*` variables are specified outside the repo at CircleCI?
Currently we have `BIOCONDA_UTILS_TAG == 'master'`. I think it would be better do use the commit rather than the tag. Otherwise we'd reuse and outdated cache when `bioconda-utils` has been changed since the last `save_cache`.
If we aren't able to directly include the commit's checksum, maybe it's possible to workaround that with something like
```bash
run:
  command: 'git ls-remote https://github.com/bioconda/bioconda-utils.git {{ .Environment.BIOCONDA_UTILS_TAG }} >bioconda-utils-tag-commit'
restore_cache:
  keys:
    - miniconda-{{ .Environment.MINICONDA_VER }}-{{ checksum "bioconda-utils-tag-commit" }}-{{ arch }}
```
?

Reply:
Good point. Maybe we should even properly tag bioconda-utils releases again. This should also increase the reliability. I will move to the latest release for now.

Reply:
Done.